src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx

```diff
@@ -82,18 +86,19 @@ interface IState {
 @replaceableComponent("views.settings.tabs.user.SecurityUserSettingsTab")
 export default class SecurityUserSettingsTab extends React.Component<IProps, IState> {
     private dispatcherRef: string;
+    private notificationState: ListNotificationState;
 
     constructor(props: IProps) {
         super(props);
 
         // Get number of rooms we're invited to
-        const invitedRooms = this.getInvitedRooms();
+        this.notificationState = RoomNotificationStateStore.instance.getListState(DefaultTagID.Invite);
```

Comment:
TBCH, using the `RoomNotificationStateStore` for this feels a little odd

Reply:
That's true, maybe `RoomListStore` would be more logical here?

Reply:
Probably, though I think it might be best to use the js-sdk directly (though you would need to filter out spaces), other people might view this differently